data/datapackage.yaml

```diff
@@ -0,0 +1,338 @@
+# --------
+# metadata: annotations.csv references.csv library.csv insertions.csv
+# --------
+
+profile: data-package
+resources:
+  - encoding: utf-8-sig
+    format: csv
+    hashing: md5
+    name: annotations
+    path: annotations.csv
+    profile: tabular-data-resource
+    description: A collection of all written annotations in Jacques Derrida's library
+    schema:
+      primaryKey: id
+      foreignKeys:
+        - fields: book_id
+          reference: 
+            resource: library
+            fields: id
+      fields:
+        - name: id
+          type: string
+          description: Unique identifier linking to the annotation's page
+          format: uri
+          constraints:
+            required: true
+            unique: true
+        - name: book_id
+          type: string
+          format: uri
+          description: Link to the book where the annotation was found
+        - name: book_title
+          type: string
+          description: Title of the book where the annotation was found
+        - name: book_type
+          type: string
+          description: Whether the work is a full book or a section of a book. 
+          constraints:
+            pattern: "Book|Book Section"
```

Comment:
journal article?

Reply:
There were no journal articles in this column, I checked when creating the yaml and it would have raised an error. Did we get the filtering wrong or does it make sense that it's not here?

Reply:
ah, yes, I wasn't paying enough attention to context (and assuming these should be same across all the datasets)

no annotated journal articles because we only digitized books